javaslang/src-gen/test/java/javaslang/CheckedFunction2Test.java

```diff
@@ -165,15 +165,6 @@ public void shouldLiftTryPartialFunction() {
         assertThat(unknown.getCause().getMessage()).isEqualToIgnoringCase("Unknown MessageDigest not available");
     }
 
-    private static final CheckedFunction2<Integer, Integer, Integer> recurrent1 = (i1, i2) -> i1 <= 0 ? i1 : CheckedFunction2Test.recurrent2.apply(i1 - 1, i2) + 1;
```

Comment:
This PR #513 shows that we should avoid `ConcurrentHashMap`
Also I believe we should keep recursive memoization if it possible